packages/jaeger-ui/src/components/Monitor/ServicesView/serviceGraph.test.js

```diff
@@ -108,35 +108,17 @@ describe('<ServiceGraph>', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('Crosshair mouse hover test', () => {
+  it('Tooltip hover test', () => {
     wrapper.setProps({
       ...props,
       loading: false,
       metricsData: {
         ...serviceMetrics.service_call_rate,
         metricPoints: [{ x: 1631271783806, y: null }, ...serviceMetrics.service_call_rate.metricPoints],
       },
-      crosshairValues: [
-        {
-          key: 1,
-          ...serviceMetrics.service_call_rate.metricPoints[0],
-        },
-      ],
-    });
-    expect(wrapper.find('AreaSeries').render()).toMatchSnapshot();
-    expect(wrapper.find('LineSeries').render()).toMatchSnapshot();
-  });
-
-  it('AreaSeries onNearestX, XYPlot onMouseLeave', () => {
-    wrapper.setProps({
-      ...props,
-      loading: false,
-      metricsData: serviceMetrics.service_call_rate,
     });
-    wrapper.find('AreaSeries').at(0).prop('onNearestX')({ x: 1, y: 2 }, { index: 7 });
-    expect(wrapper.state().crosshairValues).toEqual([{ label: 0.95 }]);
-    wrapper.find('XYPlot').prop('onMouseLeave')();
-    expect(wrapper.state().crosshairValues).toEqual([]);
```

Comment:
I think removing these tests defeats the purpose of test-driven development. Can we add some similar tests for tooltip hover test to verify the functionality?

Reply:
Yes, on it.